assets/sass/_groups.sass

```diff
@@ -0,0 +1,35 @@
+.switch
+  margin-top: 10px
+  margin-bottom: 10px
+  margin-right:5px
+
+.group
+  margin-bottom: 20px
+  margin-right: 7%
+  font-size: 16px
+  +shadow($gray)
+  border-top: 1px solid $gray
+  word-break: break-all
+  cursor: pointer
+
+  .title
+    font-size: 20px
+    font-weight: 600
+    margin-top: 5px
+    margin-bottom: 5px
+
+  .text
+    text-indent: 5px
+
+  .date
+    font-size: 14px
+    font-style: italic
+
+  .date:before
+    content: "Updated at: "
```

Comment:
what if I need to display date without such label? maybe it's a part of jade layout? or specific `.updated-date` class case